content/tracing/product_specs/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Product Specs
```

Comment:
I think we still need a better name.  These aren't quite specs.  Maybe just "More APM Info"?

Also let's move some pages from the secret FAQ to here:
- Apdex
- [Monitoring Resources](https://help.datadoghq.com/hc/en-us/articles/115003884112-How-to-create-a-Monitor-over-every-Resource-APM)

Reply:
👌